packages/manifest-plugin/manifest-overrides/common/icons.ts

```diff
@@ -1,20 +1,22 @@
-import path from 'path'
-import {type Manifest} from '../../types'
-import getFilename from '../../helpers/getFilename'
-
-const getBasename = (filepath: string) => path.basename(filepath)
-
-export default function icons(manifest: Manifest, exclude: string[]) {
-  return (
-    manifest.icons && {
-      icons: Object.fromEntries(
-        Object.entries(manifest.icons).map(([size, icon]) => {
-          return [
-            size,
-            getFilename(`icons/${getBasename(icon)}`, icon, exclude)
-          ]
-        })
-      )
-    }
-  )
-}
+import path from 'path'
+import {type Manifest} from '../../types'
+import getFilename from '../../helpers/getFilename'
+import utils from '../../helpers/utils'
+
+const getBasename = (filepath: string) => path.basename(filepath)
+
+export default function icons(manifest: Manifest, exclude: string[]) {
+  return (
+    manifest.icons && {
+      icons: Object.fromEntries(
+        Object.entries(manifest.icons).map(([size, icon]) => {
+          const outputpath = utils.unixify(getFilename(`icons/${getBasename(icon)}`, icon, exclude));

```

Comment:
utils.unixify seems relevant to all places where `getFilename` is applied, what do you think?

Reply:
It appears that all instances where `getFilename` is used require adjustment. Given my limited familiarity with the project, I'm unable to conduct a thorough verification. Could you offer me some guidance on how to proceed?

Reply:
Sure, no problem!

The manifest plugin is responsible for routing the asset paths generated by other plugins to the manifest file, so the logic that applies to `icons` also applies to other manifest fields like `background.scripts` or `chrome_url_overrides` for example.

If your changes in this PR did fix the icons path on Windows, it's likely that other manifest fields are broken as well, but not reported to us because they are not defined in the starter templates. Since `getFilename` applies to all manifest fields where a path is expected, I think moving your logic to there could fix all occurencies at once.

Something like:

import path from 'path'
import utils from './utils'

```diff
// manifest-plugin/helpers/getFilename.ts

export default function getFilename(
  feature: string,
  filepath: string,
  exclude: string[]
) {
  const entryExt = path.extname(filepath)

  // Do not attempt to rewrite the asset path if it's in the exclude list.
  const shouldSkipRewrite = utils.shouldExclude(filepath, exclude)

  const fileOutputpath = shouldSkipRewrite ? path.normalize(filepath) : feature

  if (['.js', '.jsx', '.tsx', '.ts'].includes(entryExt)) {
    return fileOutputpath.replace(entryExt, '.js')
  }

  if (['.html', '.njk', '.nunjucks'].includes(entryExt)) {
    return fileOutputpath.replace(entryExt, '.html')
  }

  if (['.css', '.scss', '.sass', '.less'].includes(entryExt)) {
    return fileOutputpath.replace(entryExt, '.css')
  }

- return fileOutputpath
+ return utils.unixify(fileOutputpath)
}
```

Reply:
I updated the test case and need your confirmation.